tests/rkt_net_test.go

```diff
@@ -82,15 +82,15 @@ func TestNetHostConnectivity(t *testing.T) {
 	ctx := newRktRunCtx()
 	defer ctx.cleanup()
 
-	cmd := fmt.Sprintf("%s --net=host --debug --insecure-skip-verify run --mds-register=false %s", ctx.cmd(), testImage)
-	child := spawnOrFail(t, cmd)
-
 	ga := testutils.NewGoroutineAssistant(t)
+
 	// Child opens the server
+	cmd := fmt.Sprintf("%s --net=host --debug --insecure-skip-verify run --mds-register=false %s", ctx.cmd(), testImage)
+	child := ga.SpawnOrFail(cmd)
```

Comment:
That will deadlock if spawning fails - it will try to send an error over a blocking channel which is not listened yet by anyone.